website/templates/emails/comment_replies.txt.mako

```diff
@@ -1,3 +1,10 @@
 ${user} replied to your comment "${parent_comment}" on your project "${title}": ${content}.
 
-${'\t'}To view this on the Open Science Framework, please visit: ${url}.
\ No newline at end of file
+${'\t'}To view this on the Open Science Framework, please visit: ${url}.
+
+
+Center for Open Science
+210 Ridge McIntire Road
+Suite 500
+Charlottesville, VA 22903-5083
+Privacy Policy: https://github.com/CenterForOpenScience/cos.io/blob/master/PRIVACY_POLICY.md
```

Comment:
Driveby comment: 
The address/privacy policy is exactly the same in most of these (aside from some with formatting). 
Is there a way this could be made into a variable and put into something like `language.py` in case other changes to the signature are required (e.g., we move or change the location of the Privacy Policy)? 

Just a idea, might be worth looking into.